packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -676,6 +676,10 @@ export const guides = [
             title: "Email notifications",
             guideName: "custom-email",
           },
+          {
+            title: "Migrate brand customizations",
```

Comment:
<img width="360" height="318" alt="image" src="https://github.com/user-attachments/assets/eae86b0b-4a56-461c-aa00-a97d3975776f" /> "Migrate" appears earlier in the same menu to mean "upgrade the version of". We're using it here for "move between orgs". This might be confusing?